.devcontainer/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM node:20
+
+ARG TZ
+ENV TZ="$TZ"
+
+RUN apt update && apt install -y less git procps sudo fzf zsh man-db unzip gnupg2 gh iptables ipset iproute2 dnsutils aggregate jq rsync
```

Hadolint (2.12.0)

[warning] Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead

(DL3027)